### DISCUSSION TITLE
Is it possible to disable wordpress and joomlatools pages install and only use joomla?

### DISCUSSION THREAD

**Discussion by alexandreelise** [joomlatools/joomlatools-pages-server#98] [Q&A] [answered]
In joomlatools server. Is it possible to disable wordpress and joomlatools pages install and only use joomla?

**Comment by johanjanssens** [comment, marked as answer]
Both automated Wordpress and Joomla installs can easily be disabled by changing your local .env file. The default file we added sets up both Joomla and Wordpress mostly for testing purposes while we are working on server. Jus remove

```
WORDPRESS_SITE=wordpress
WORDPRESS_VERSION=latest
```

From https://github.com/joomlatools/joomlatools-server/blob/master/.env to disable installing Wordpress, or you can remove both the default Joomla and Wordpress installs and use `jtctl` to create sites. See: https://github.com/joomlatools/joomlatools-server/pull/71

Pages is automatically installed as it's used for the internal dashboard which us being worked on in #55. The dashboard is running Pages in standalone mode. It's not automatically installed in Joomla.

**Reply by alexandreelise** [reply]
Hi Johan, you and your team are effectively and efficiently reusing all the tools you created along the way. A unified joomlatools ecosystem building blocks. It must have been hard to plan that masterplan years ago @johanjanssens.

**Reply by johanjanssens** [reply]
Thanks. One of the reasons to build pages and make it run standalone was to be able to replace the dashboard in our Vagrant box with a more dynamic solution. This has been in the wishlist of a few years. Lots of other reasons too, but thats definitely one of them.
